package/MDAnalysis/coordinates/PDB.py

```diff
@@ -236,6 +236,7 @@ def __init__(self, filename, **kwargs):
 
         try:
             self._n_atoms = kwargs['n_atoms']
+            self.n_atoms = self._n_atoms # ?? does this have to be private?
```

Comment:
Depends. To we demand `n_atoms` to be passed to the init function? If we demand it can can be public. Otherwise I would make it private and a `n_atoms` property that guesses it from the PDB itself.